src/Stratis.Bitcoin.Features.Miner/MinerSettings.cs

```diff
@@ -80,6 +98,11 @@ public MinerSettings(NodeSettings nodeSettings)
 
             this.BlockDefinitionOptions = new BlockDefinitionOptions(blockMaxWeight, blockMaxSize).RestrictForNetwork(nodeSettings.Network);
 
+            this.EnableCoinStakeSplitting = config.GetOrDefault("enablecoinstakesplitting", true, this.logger);
+            this.MinimumSplitCoinValue = config.GetOrDefault("minimumsplitcoinvalue", (ulong) (100 * Money.COIN) , this.logger);
```

Comment:
Somehow a lot of people started to use this space between `(type) (value)` but original code elsewhere does not have this space -> `(type)(value)`

Same on the next line